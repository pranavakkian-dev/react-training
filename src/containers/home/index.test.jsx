import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import Home from './index';
import Store from '../../redux/store';

describe('Home', () => {
  test('Component rendered successfully!', () => {
    const { container } = render(
      <Provider store={Store}>
        <Home />
      </Provider>,
    );
    expect(container).toMatchSnapshot();
  });
});
