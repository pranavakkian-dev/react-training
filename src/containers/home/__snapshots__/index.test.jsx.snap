// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component rendered successfully! 1`] = `
<div>
  <h1>
    Welcome,
     
  </h1>
</div>
`;
